docs/configuration.rst

```diff
@@ -156,6 +156,18 @@ instead of ``entity-manager``.
     files, the last one will be used. So you have to list all the listeners you
     want to detach.
 
+Create table
+----------------------------------
+To be able to use the bundle with default configuration, an ``ext_log_entries`` table must be created. This can be done with doctrine migrations facilities:
```

Comment:
This is only needed if you enable certain featured, like loggable or translatable. I would move this section up to the bottom of the `Add the extensions to your mapping` section. And probably we should just mention something like:

> If you added any of these mappings, be sure to update your schema to add the new table(s) - i.e. by
> generating and executing a migration.

The user may not have DoctrineMigrationsBundle installed, which is why the more generic language is probably better.